backend/src/services/person.service.ts

```diff
@@ -9,9 +9,15 @@ const createPerson = async (personDetails: PersonModel) => {
   return person;
 };
 
-const getPersonWithId = async (reqPersonID: string) => {
-  return await Person.findOne({_id: reqPersonID});
-}
+const updatePersonWithId = async (reqPersonId: string, personNewDetails: PersonModel) => {
+  const query = { _id: reqPersonId };
+  return Person.findOneAndUpdate(query, personNewDetails, { upsert: true });
```

Comment:
Might want to change `findOneAndUpdate` to `findByIdAndUpdate` as used in Hajin's PR for updating encounters.

Reply:
I don't think this will impact performance much since _id is still the thing being indexed in the search